### PR TITLE
II-2829 Update Prometheus agent

### PR DESCRIPTION
**Pull request by moharnab123saikia**

### CODE REVIEW

File: deployment/DeployAgent/inventory [hunk 1]
```diff
@@ -109,6 +109,14 @@ cadvisorPort=8080
 ##Vars for NFSen/NFDump
 nfsenProfile=/data/nfsen/profiles-data/live
 
+## metric grouping
```
Comment (ptmr3) [inline deployment/DeployAgent/inventory]: If this is for Prometheus, it should be tagged as such

File: prometheus/getmetrics_prometheus.py [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+    if response.status_code == 200:
+        logger.info(str(len(bytearray(to_send_data_json))) +
+                    " bytes of data are reported.")
+    else:
+        logger.info("Failed to send data.")
+    logger.debug("--- Send data time: %s seconds ---" %
+                 (time.time() - send_data_time))
+
+
+def chunks(l, n):
```
Comment (ptmr3) [inline prometheus/getmetrics_prometheus.py]: rename l, n

File: prometheus/getmetrics_prometheus.py [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+    logger_obj.addHandler(logging_handler_out)
+
+    logging_handler_err = logging.StreamHandler(sys.stderr)
+    logging_handler_err.setLevel(logging.WARNING)
+    logger_obj.addHandler(logging_handler_err)
+    return logger_obj
+
+
+if __name__ == "__main__":
+    GROUPING_START = 15000
```
Comment (ptmr3) [inline prometheus/getmetrics_prometheus.py]: isnt this supposed to come from config?

File: opentsdb/getmetrics_opentsdb_oldapi.py [hunk 4]
```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python
```
Comment (ptmr3) [inline opentsdb/getmetrics_opentsdb_oldapi.py]: this should not be in this PR

File: prometheus/getmetrics_prometheus.py (hunk 3 above)
Comment (moharnab123saikia) [reply prometheus/getmetrics_prometheus.py]: The start range is fixed for an agent type. We take this as base

**Pull request merged**